tests/release/pipelines/e2e-fbc.go

```diff
@@ -0,0 +1,166 @@
+package pipelines
+
+import (
+	"fmt"
+	"time"
+
+	ecp "github.com/enterprise-contract/enterprise-contract-controller/api/v1alpha1"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	appservice "github.com/redhat-appstudio/application-api/api/v1alpha1"
+	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils"
+	releaseApi "github.com/redhat-appstudio/release-service/api/v1alpha1"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+)
+
+var _ = framework.ReleasePipelinesSuiteDescribe("[HACBS-2385] e2e test for fbc happy path.", Label("release-pipelines", "fbcHappyPath"), func() {
+	defer GinkgoRecover()
+
+	var err error
+	var devWorkspace = "dev-release-team"
+	var managedWorkspace = "managed-release-team"
+	var devNamespace = devWorkspace + "-tenant"
+	var managedNamespace = managedWorkspace + "-tenant"
+
+	var fbcApplicationName = "fbc-kas-app"
+	var fbcComponentName = "fbc-kas-comp"
+	var fbcReleasePlanName = "fbc-kas-releaseplan"
+	var fbcReleasePlanAdmissionName = "fbc-kas-releaseplanadmission"
+	var fbcReleaseStrategyName = "fbc-kas-strategy"
+	var fbcEnterpriseContractPolicyName = "fbc-kas-policy"
+	var fbcServiceAccountName = "release-service-account"
+	var fbcSourceGitUrl = "https://github.com/redhat-appstudio-qe/fbc-sample-repo"
+	var releasePipelineRunCompletionTimeout = 20 * time.Minute
+	var releaseCreationTimeout = 5 * time.Minute
+	var defaultInterval = 100 * time.Millisecond
+
+	var component *appservice.Component
+	var releaseCR *releaseApi.Release
+	var buildPr *v1beta1.PipelineRun
+	var releasePr *v1beta1.PipelineRun
+	var snapshot *appservice.Snapshot
+
+	stageOptions := utils.Options{
+		ToolchainApiUrl: utils.GetEnv("RHTAP_TOOLCHAIN_API_URL", ""),
+		KeycloakUrl:     utils.GetEnv("RHTAP_KEYLOAK_URL", ""),
+		OfflineToken:    utils.GetEnv("RHTAP_OFFLINE_TOKEN", ""),
+	}
+
+	dev_fw, err := framework.NewFrameworkWithTimeout(
+		devWorkspace,
+		time.Minute*60,
+		stageOptions,
+	)
+	Expect(err).NotTo(HaveOccurred())
+
+	managed_fw, err := framework.NewFrameworkWithTimeout(
+		managedWorkspace,
+		time.Minute*60,
+		stageOptions,
+	)
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
please move this code to the `BeforeAll` node

Reply:
done